assignments/function-conversion.js

```diff
@@ -3,28 +3,38 @@
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
-// myFunction();
+let myFunction = () => {console.log("Function was invoked!");};
```

Comment:
here as well, you can use `const` instead of `let`, prioritize `const` first and use `let` when you need to.